ClaudiaHernandezGarcia/springBootProject/src/main/java/com/globant/springBootProject/service/impl/AnagramsServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.globant.springBootProject.service.impl;
+
+import com.globant.springBootProject.dto.AnagramsDTO;
+import com.globant.springBootProject.service.AnagramsService;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class AnagramsServiceImpl implements AnagramsService {
+
+    @Override
+    public AnagramsDTO Solution(AnagramsDTO anagramsDTO) {
```

Comment:
Solution??? uppercase for a method name??
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names